filebeat/module/nginx/_meta/docs.asciidoc

```diff
@@ -0,0 +1,45 @@
+== Nginx module
+
+This module parses access and error logs created by the http://nginx.org/[Nginx]
+HTTP server.
+
+[float]
+=== Compatibility
+
+The Nginx module was tested with logs from version 1.10.
+
+On Windows, the module was tested with Nginx installed from the Chocolatey
+repository.
+
+[float]
+=== Dashboard
+
+This module comes with a sample dashboard.
+
+image::./images/kibana-nginx.png[]
+
+[float]
+=== Access logs fileset settings
+
+[float]
+==== var.paths
```

Comment:
As this exists for all modules / filesets, should we describe it somewhere global?